.github/workflows/deploy.yml

```diff
@@ -0,0 +1,55 @@
+name: DEPLOY
+on:
+  workflow_dispatch:
+    permissions:
+      teams:
+        - 'reactor/team'
+    inputs:
+      application:
+        description: 'Application name to deploy:'
+        required: true
+        type: choice
+        default: 'projectreactor-test'
+        options:
+          - 'projectreactor-test'
+          - 'projectreactor'
+
+permissions: read-all
+
+jobs:
+  deploy-cf:
+    runs-on: reactor-self-hosted
+    env:
+      APPNAME: ${{ github.event.inputs.application }}
+      ROUTE: ${{ github.event.inputs.application == 'projectreactor' && 'https://projectreactor.io' || 'https://projectreactor-test.wdc-08-pcf1-pws-apps.oc.vmware.com' }}
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+        with:
+          fetch-depth: 0 #for spotless ratchet
+      - uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4
+        with:
+          distribution: 'temurin'
+          java-version: 8
```

Comment:
if you don't mind, let's try to reduce as much as possible the changes. As said, there may be more PRs to submit before it can work, so I would rather prefer to do revisit using separate future PRs.

Reply:
Please add somewhere a TODO list

Reply:
created #132